django_celery_extensions/management/commands/run_celery.py

```diff
@@ -37,19 +51,22 @@ def add_arguments(self, parser):
         )
 
     def handle(self, *args, **options):
+        celery_type = options.get('type')
+        celery_settings = options.get('celery_settings')
+        extra_arguments = options.get('extra_args', '')
         if options.get('use_reloader'):
-            self.stdout.write('Starting celery with autoreload...')
+            if os.environ.get(DJANGO_AUTORELOAD_ENV) != 'true':
```

Comment:
django autoreload is weird thing. If the run_with_reloader is called it will start the same command in subprocess with new env parameter DJANGO_AUTORELOAD_ENV set to true. The subprocess check this env and starts code changes checking. If something is changed the subprocess is killed and is started again. If I will remove this if the stdout is writeen twice. 

Next think is that if sigterm is performed the subprocess is not stopped unfortunatelly. I should set there sigterm signal but django will override it with sys.exit. I have no way how to do it, only rewrite run_with_reloader and this is ugly. Therefore is better ignore it. This is only dev thing and you are using ctrl + C not sigterm and it works well.